src/command_modules/azure-cli-role/azure/cli/command_modules/role/_params.py

```diff
@@ -55,8 +56,13 @@ def load_arguments(self, _):
     with self.argument_context('ad app permission') as c:
         c.argument('api_permissions', nargs='+', help='space seperated list of <resource-access-id>=<type>')
         c.argument('expires', help='Expiry date for the permissions in years. e.g. 1, 2 or "never"')
-        c.argument('scope', help='oauth scope')
+        c.argument('scope', help='Specifies the value of the scope claim that the resource application should expect in the OAuth 2.0 access token, e.g. User.Read')
```

Comment:
Do people generally know what valid scopes look like? Are there enums/docs?

Reply:
No. Scope are defined by each specific web-api, so users will have to get to their docs to see. For AAD Graph api, users usually check out the manifest file in portal and know the value to use.
It is not a very pleasant experience, but our commands now at least get the scripting possible for permission manage end to end.